examples/plot_simulate_alpha.py

```diff
@@ -29,31 +29,24 @@
 print(params)
 
 ###############################################################################
-# Remove all evoked proximal feed parameters
-for key in params['*evprox*']:
-    del params[key]
-
-# Remove all evoked distal feed parameters
-for key in params['*evdist*']:
-    del params[key]
-
-###############################################################################
-# Now, we update a few rhythmic feed parameters
+# Update a few of the default parameters related to visualisation
 params.update({
     'dipole_scalefctr': 150000.0,
     'dipole_smooth_win': 0,
     'tstop': 310.0,
-    't0_input_dist': 50.0,
-    'tstop_input_dist': 1001.0,
-    'input_dist_A_weight_L2Pyr_ampa': 5.4e-5,
-    'input_dist_A_weight_L5Pyr_ampa': 5.4e-5,
-    'sync_evinput': 1,
-    "prng_seedcore_input_dist": 3
 })
 
 ###############################################################################
 # Now let's simulate the dipole and plot it
 net = Network(params)
+
+weights_ampa = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+net.add_bursty_drive(
+    'bursty', distribution='normal', t0=50., sigma_t0=0., T=params['tstop'],
+    burst_f=10, spike_jitter_std=20., numspikes=2, spike_isi=10, repeats=10,
```

Comment:
would it be too difficult to make the following changes:

burst_f -> burst_freq
t0 -> tstart
T -> tstop

in MNE, we have tried to stay away from single letter variable names and such because they are hard to grep.

Reply:
Cool beans. Could it be a smaller PR after this merge?

Reply:
sure thing!

Reply:
Also, `spike_jitter_std` represents variance in the burst event time rather than the inter-burst spike time right? If so, what do you think of renaming the arg to something simpler like `burst_std`?

Reply:
I've gone back and forth on this myself. The variable is used like this:

```python
t_array = prng.normal(np.repeat(isi_array, repeats), spike_jitter_std)
```

Do you still feel `burst_std` is appropriate? The intra-burst ISI is fixed, this parameter takes the `isi_array` and jitters each fixed step. Note also the `repeats` (I've seen 10 used there), which adds to the confusion...

Reply:
The way I read this (and the way the HNN documentation presents it) is that each burst consists of either 1 or 2 spikes. If there are 2 or more spikes/burst, a fixed ISI determines the time between each spike. `t_array` jitters the location of each burst across repeats so that it simulates the arrival time of multiple (i.e., `repeats=10`) bursty processes arriving from thalamus. To that end, I think that `isi_array` has a misleading name since it does not contain ISI values but rather time values corresponding to the center of each burst. After `t_array` is defined, the `events_per_cycle` variable is used to assign multiple spikes with fixed ISI centered at the burst location. Does that sound correct?

Reply:
That's correct. Note that there's both a `repeats` _and_ a `tstop`. So the former isn't 'repeating' the cycle 10 times, but rather emulating 10 almost-simultaneous burst packets arriving on _each_ cycle.

But do you still feel `burst_std` covers it? It's more like `burst_center_std`, but maybe that's too clunky?

Reply:
@rythorpe I think your suggestion is actually most accurate: the STD applies to the burst "centers", not individual spikes in a burst. How about:

* `burst_rate`: The mean rate at which cyclic bursts occur (unit: Hz)
* `burst_std`: The standard deviation of the burst occurrence on each cycle (unit: ms). Only applied when the burst distribution is 'normal'.
* `numspikes`: The number of spikes in a burst
* `spike_isi`: The inter-spike-interval (unit: ms)
* `repeats`:  The number of bursts per cycle

@jasmainak  if we agree on these, I'll change

* `t0` -> `tstart`
* `T` -> `tstop`

throughout?

Reply:
Resolving, implemented in: 3f81f68